src/init/mod.rs

```diff
@@ -117,9 +117,18 @@ pub(crate) fn init_rootfs() {
         return;
     }
 
-    mount_vfs("none", "/dev", "devtmpfs").unwrap();
-    mount_vfs("none", "/sys", "sysfs").unwrap();
-    mount_vfs("proc", "/proc", "proc").unwrap();
+    let r = mount_vfs("none", "/dev", "devtmpfs");
+    if r.is_err() {
+        panic!("unable to mount /dev");
```

Comment:
since you are panicking anyway, it makes sense to simply replace unwrap with expect. it allows you to still provide an error message and save some lines

Reply:
You are right @bpmooch, but I'm going to iterate on this anyway